dd-java-agent/appsec/src/main/java/com/datadog/appsec/AppSecSystem.java

```
@@ -156,7 +156,7 @@ private static void loadModules(EventDispatcher eventDispatcher) {
       }
 
       for (AppSecModule.EventSubscription sub : module.getEventSubscriptions()) {
-        eventSubscriptionSet.addSubscription(sub.eventType, sub);
+        eventSubscriptionSet.setSubscription(sub.eventType, sub);
```

It will be nice if we could log a warn/error message in case there are multiple subscribers, otherwise they will be silently ignored.